arch/src/aarch64/mod.rs

```diff
@@ -3,16 +3,26 @@
 
 pub mod layout;
 
```

Comment:
For 24e62df7b3f93de8127f15d0a7a08e552bfccfd9 please prefix with "build:" and drop gerrit Change-Id.